test/libsolidity/util/TestFileParser.h

```diff
@@ -131,10 +132,18 @@ class TestFileParser
 	/// empty. If so, the signature is not allowed to define any parameters.
 	std::pair<std::string, bool> parseFunctionSignature();
 
+	/// FunctionValue structure
+	/// Holds value along with coin it is expressed in
+	struct FunctionValue
```

Comment:
Why not also use this in the FunctionCall struct?

Reply:
I completely agree, doing it now